workflows/dfe.snake

```diff
@@ -317,7 +317,7 @@ rule plot_results:
             dadi_bestfits = [ b for b in dadi_bestfits if 'pop0' in b ]
             polydfe_bestfits = [ b for b in polydfe_bestfits if 'pop0' in b ]
             dfe_alpha_bestfits = [ b for b in dfe_alpha_bestfits if 'pop0' in b ]
-            xtick_label = None
+            xtick_label = ['pop0']
```

Comment:
why not get these population IDs directly from the demographic model? it should be something like
```suggestion
            xtick_label = species.get_demographic_model(wildcards.demog).populations[popID].name
```

Reply:
ok